Test failure (git authentication) for `datalad_fuse/tests/test_fsspec_head.py::test_git_repo` on `0.5.1`

Full build and test log: https://ppb.chymera.eu/cb592e.html

Error excerpt:

```
E               subprocess.CalledProcessError: Command '['git', 'commit', '-m', 'Add a file']' returned non-zero exit status 128.

capture_output = False
check      = True
input      = None
kwargs     = {'cwd': PosixPath('/var/tmp/portage/dev-vcs/datalad-fuse-0.5.1/temp/pytest-of-portage/pytest-0/test_git_repo0')}
popenargs  = (['git', 'commit', '-m', 'Add a file'],)
process    = <Popen: returncode: 128 args: ['git', 'commit', '-m', 'Add a file']>
retcode    = 128
stderr     = None
stdout     = None
timeout    = None

/usr/lib/python3.11/subprocess.py:571: CalledProcessError
----------------------------------------------------------- Captured stdout call -----------------------------------------------------------
Initialized empty Git repository in /var/tmp/portage/dev-vcs/datalad-fuse-0.5.1/temp/pytest-of-portage/pytest-0/test_git_repo0/.git/
----------------------------------------------------------- Captured stderr call -----------------------------------------------------------
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'portage@localhost.(none)')
```

I'm trying to solve this now on the gentoo sandbox side, but maybe the software shouldn't assume it's being tested in a pre-configured personal environment?

--- a/.github/workflows/docbuild.yml
+++ b/.github/workflows/docbuild.yml
@@ -8,10 +8,6 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Set up environment
-      run: |
-        git config --global user.email "test@github.land"
-        git config --global user.name "GitHub Almighty"
     - uses: actions/checkout@v4
     - name: Set up Python 3.8
       uses: actions/setup-python@v5

--- a/.github/workflows/test-libfuse.yml
+++ b/.github/workflows/test-libfuse.yml
@@ -20,11 +20,6 @@ jobs:
           sudo apt-get install eatmydata
           sudo eatmydata apt-get install -y fuse git-annex-standalone
 
-      - name: Set up environment
-        run: |
-          git config --global user.email "test@github.land"
-          git config --global user.name "GitHub Almighty"
-
       - uses: actions/checkout@v4
 
       - name: Set up Python 3.8

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -19,10 +19,6 @@ jobs:
         sudo apt-get update -qq
         sudo apt-get install eatmydata
         sudo eatmydata apt-get install git-annex-standalone
-    - name: Set up environment
-      run: |
-        git config --global user.email "test@github.land"
-        git config --global user.name "GitHub Almighty"
     - uses: actions/checkout@v4
     - name: Set up Python 3.8
       uses: actions/setup-python@v5

--- a/datalad_fuse/tests/test_fsspec_head.py
+++ b/datalad_fuse/tests/test_fsspec_head.py
@@ -126,7 +126,7 @@ def test_subdataset_get_bytes_binary(superdataset):
     )
 
 
-def test_git_repo(tmp_path):
+def test_git_repo(tmp_path, tmp_home):  # noqa: U100
     subprocess.run(["git", "init"], cwd=tmp_path, check=True)
     TEXT = (Path(__file__).with_name("data") / "text.txt").read_bytes()
     (tmp_path / "text.txt").write_bytes(TEXT)

